api_client/python/timesketch_api_client/sketch.py

```diff
@@ -249,7 +249,7 @@ def upload(self, timeline_name, file_path):
         """
         resource_url = '{0:s}/upload/'.format(self.api.api_root)
         files = {'file': open(file_path, 'rb')}
-        data = {'name': timeline_name, 'sketch_id': self.id}
+        data = {'name': timeline_name, 'sketch_id': self.id, 'index_name': index}
```

Comment:
api_client/python/timesketch_api_client/sketch.py:252:0: C0301: Line too long (81/80) (line-too-long)

the linter is complaining here,